### ISSUE TITLE
Compile before saving PDF

### ISSUE THREAD

**Issue by dingens** [alexandervdm/gummi#73]
Option Preferences → Misc → _Automatically export PDF on document save_ is active. The user makes some changes to the document, then immediately saves the document.

Expected behavior: The exported PDF document contains the recently added contents.

Actual behavior: The exported PDF represents the state of the document at the time of the last compilation.

Proposed change: After saving the document, first re-compile the document, then save the PDF.

**Comment by oskar-skog** [comment]
Is this still present in the current version?

**Comment by dingens** [comment]
Yes. When I enter some text and press Ctrl-S immediately, the new text is not inside the pdf.
